Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Revenue.hs

```diff
@@ -59,10 +58,20 @@ data AllFees = AllFees
   }
   deriving (Generic, Show, FromJSON, ToJSON, ToSchema)
 
-data CashCollectionListRes = CashCollectionListRes
-  { summary :: Summary,
+data CollectionListRes = CollectionListRes
+  { totalCount :: Int,
     totalFeeCollected :: HighPrecMoney,
-    collectedFees :: [DriverFeeAPIEntity]
+    totalRides :: Int,
+    totalDrivers :: Int,
+    collectionsTs :: [(HighPrecMoney, Maybe UTCTime)],
+    numRidesTs :: [(Int, Maybe UTCTime)],
+    paymentTs :: [(Int, Maybe UTCTime)]
```

Comment:
`collectionsTs` and `numRidesTs` also require grouping like `paymentTs`

Reply:
no think , it will work.

Reply:
No, it can have entries with the same timestamp. You need to group it and add money/rides with same timestamp